src/test/TestSVG.java

```diff
@@ -27,6 +40,65 @@ public void createSVG_fail(){
 		f1.writeSVGFile();
 		SVGFile f2 = new SVGFile("testFile","some content");
 		Assert.assertEquals(false, f2.writeSVGFile());
+		f1.getFile().delete();
+	}
+	
+	/************************************
+	 *	Paths with one Bezier Line
+	 ************************************/
+	@Test
+	public void create_open_path(){
+	Pencil pred = new Pencil(3, new ColorSVG(Color.red.getRGB()));
+	Path path2 = new Path();
+	path2.setStart(new Point(100, 350));
+	path2.setPencil(pred);
+	ArrayList<PathPart> parts2 = new ArrayList<PathPart>();
+	parts2.add(new PathPart(new Point(250, 50), LineType.Line));
+	path2.setParts(parts2);
+	
```

Comment:
Add comments at the beginning of each paragraph, saying that you add a Line, a QuadractiBezier ... etc